### PR TITLE
Add metadata to list in m365 adoption gallery

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | no                                |
| New sample?     | no                                |
| Related issues? | NA |

## What's in this Pull Request?

This adds sample.json files to existing samples to ensure they are listed at https://adoption.microsoft.com/en-us/sample-solution-gallery/

### CODE REVIEW

File: Samples/office-add-in-commands/powerpoint/assets/sample.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[
+    {
+      "name": "powerpoint-add-in-commands",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Create a PowerPoint add-in with command buttons",
+      "shortDescription": "Create a PowerPoint add-in with command buttons.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/office-add-in-commands/powerpoint",
+      "longDescription": [
+        "Learn how to build an Office Add-in that has a command button to show the task pane, and a menu dropdown button that can show the task pane, or get data."
```
Comment (AlexJerabek) [inline Samples/office-add-in-commands/powerpoint/assets/sample.json:10]:
```suggestion
        "Learn how to build an Office Add-in that has a command button to show the task pane and a menu dropdown button that shows the task pane or gets data."
```

File: Samples/office-add-in-commands/word/assets/sample.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+[
+    {
+      "name": "word-add-in-commands",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Create a Word add-in with command buttons",
+      "shortDescription": "Create a Word add-in with command buttons.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/office-add-in-commands/word",
+      "longDescription": [
+        "Learn how to build an Office Add-in that has a command button to show the task pane, and a menu dropdown button that can show the task pane, or get data."
```
Comment (AlexJerabek) [inline Samples/office-add-in-commands/word/assets/sample.json:10]:
```suggestion
        "Learn how to build an Office Add-in that has a command button to show the task pane and a menu dropdown button that shows the task pane or gets data."
```

File: Samples/excel-create-worksheet-from-web-site/assets/sample.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+[
+    {
+      "name": "excel-add-in-create-spreadsheet-from-web-page",
```
Comment (AlexJerabek) [inline Samples/excel-create-worksheet-from-web-site/assets/sample.json:3]: Should this say "page" or "site"?

File: Samples/excel-shared-runtime-global-state/assets/sample.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+[
+    {
+      "name": "office-add-in-shared-runtime-global-data",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Share global data with a shared runtime",
+      "shortDescription": "Shows how to set up a basic Office Add-in project that uses the shared runtime.",
```
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-global-state/assets/sample.json:7]: Not sure it's a "basic" concept ;-)
```suggestion
      "shortDescription": "Shows how to set up an Office Add-in project that uses the shared runtime.",
```

File: Samples/excel-shared-runtime-scenario/assets/sample.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
+[
+    {
+      "name": "office-add-in-ribbon-task-pane-ui",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Manage ribbon and task pane UI, and run code on doc open",
+      "shortDescription": "Shows contextual ribbon buttons, hide or show task pane, and how to start add-in on document open.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/excel-shared-runtime-scenario",
```
Comment (AlexJerabek) [inline Samples/excel-shared-runtime-scenario/assets/sample.json:8]: This looks incorrect.

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
I mostly left comments trying to make the descriptions sound a little snappier. Feel free to disagree with those.

File: Samples/excel-create-worksheet-from-web-site/assets/sample.json (hunk 3 above)
Comment (davidchesnut) [reply Samples/excel-create-worksheet-from-web-site/assets/sample.json:3]: The intent is about your web site (any page on your site, or perhaps even a shared ribbon could be the source to create a spreadsheet.

File: Excel-custom-functions/web-worker/assets/sample.json [hunk 6]
```diff
@@ -0,0 +1,35 @@
+[
+    {
+      "name": "excel-custom-function-web-worker-pattern",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Custom function sample using web worker",
+      "shortDescription": "This sample shows how to use web workers in custom functions to prevent blocking the UI of your Office Add-in.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Excel-custom-functions/web-worker",
+      "longDescription": [
+        "This sample shows how to use web workers in custom functions to prevent blocking the UI of your Office Add-in."
```
[the remark this answers is not in the document]
Comment (davidchesnut) [reply Excel-custom-functions/web-worker/assets/sample.json:10]: Good point!

File: Samples/VSTO-shared-code-migration/assets/sample.json [hunk 7]
```diff
@@ -0,0 +1,46 @@
+[
+    {
+      "name": "office-add-in-migrate-vsto-to-web",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Use a shared library to migrate your VSTO add-in to an Office web add-in",
+      "shortDescription": "Use a shared library to migrate your Visual Studio Tools for Office add-in to an Office web add-in.",
```
[the remark this answers is not in the document]
Comment (davidchesnut) [reply Samples/VSTO-shared-code-migration/assets/sample.json:7]: Following the VSTO docs here, so hope they are correct: https://learn.microsoft.com/en-us/visualstudio/vsto/office-solutions-development-overview-vsto

File: Samples/excel-create-worksheet-from-web-site/assets/sample.json (hunk 3 above)
Comment (davidchesnut) [reply Samples/excel-create-worksheet-from-web-site/assets/sample.json:3]: oh the article is inconsistent. I need to reexamine the usage here and make them consistent.

File: Samples/excel-shared-runtime-scenario/assets/sample.json [hunk 8]
```diff
@@ -0,0 +1,61 @@
+[
+    {
+      "name": "office-add-in-ribbon-task-pane-ui",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Manage ribbon and task pane UI, and run code on doc open",
+      "shortDescription": "Shows contextual ribbon buttons, hide or show task pane, and how to start add-in on document open.",
```
[the remark this answers is not in the document]
Comment (davidchesnut) [reply Samples/excel-shared-runtime-scenario/assets/sample.json:7]: I like your change, but when I think of SEO it seems more likely a developer searches for "how to open the task pane" than "how to change the visibility of the task pane".

File: Samples/excel-shared-runtime-scenario/assets/sample.json [hunk 9]
```diff
@@ -0,0 +1,61 @@
+[
+    {
+      "name": "office-add-in-ribbon-task-pane-ui",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Manage ribbon and task pane UI, and run code on doc open",
```
[the remark this answers is not in the document]
Comment (davidchesnut) [reply Samples/excel-shared-runtime-scenario/assets/sample.json:6]: fixed

File: Samples/excel-shared-runtime-scenario/assets/sample.json (hunk 5 above)
Comment (davidchesnut) [reply Samples/excel-shared-runtime-scenario/assets/sample.json:8]: The url is correct and goes to Excel shared runtime scenario (unless I'm missing something?)

File: Samples/excel-create-worksheet-from-web-site/assets/sample.json (hunk 3 above)
Comment (davidchesnut) [reply Samples/excel-create-worksheet-from-web-site/assets/sample.json:3]: Page is fine. changed here.

File: Samples/excel-shared-runtime-scenario/assets/sample.json (hunk 5 above)
Comment (AlexJerabek) [reply Samples/excel-shared-runtime-scenario/assets/sample.json:8]: I guess the name confused me, but the URL matches the folder.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**